### PR TITLE
Pickup file to be used used as fallback

### PR DESCRIPTION
**Pull request by fmherschel**
* e.g. if hook is called during a phase when the cluster is stopped - in maintenance procedures
* updated man pages
* fixed typos and code indents

### CODE REVIEW

File: ra/SAPHana [hunk 1]
```diff
@@ -2216,14 +2221,55 @@ function saphana_stop_clone() {
...
+                #
+                site="${srATTR#hana_${sid}_site_srHook_}"
+                super_ocf_log info "DEC: srHookAttribute file found - recover lost SAP HA/DR event ($srATTR=$srVALUE)"
+                #
+                # DONE: PRIO0 - add check for srATTR name here
+                #
+                case "$srVALUE" in
+                SOK )
+                      super_ocf_log info "DEC: recover attributes from file $srHookAttributeFile - calling set_SRHOOK $site SOK"
+                      set_SRHOOK "$site" "SOK"
```
Comment (angelabriel) [inline ra/SAPHana:2259]: The function "set_SRHOOK" is missing. It is available in the ScaleOut RA, but currently missing here.

**Review by angelabriel — CHANGES_REQUESTED** [review]
Looks amazing, many thanks. Only a small part is missing.

File: ra/SAPHana (hunk 1 above)
Comment (fmherschel) [reply ra/SAPHana:2259]: Update will follow later.

**Comment by fmherschel** [comment]
* set_SRHOOK() added
* get_SRHOOK_plain() added (only queries the hook attribute, not the pull attribute for code parts where this was needed
* removed redundant parts to set the ATTR as a dynamic attribute definition
* improved indentation in one of the functions to improve code readability

**Comment by fmherschel** [comment]
@angelabriel Maybe you like to (again) have a look on this. The newest changes are summarized above.

**Review by angelabriel — APPROVED** [review]
Looks pretty good to me, thanks a lot.

**Pull request merged**